### PR TITLE
SPR: document more day 2 operations and trivy github token (CAPS-199, CAPS-201)

### PR DESCRIPTION
**Pull request by stefannica**
Document more Day 2 operations for SUSE Private Registry, as well as providing a github token for trivy.

### CODE REVIEW

File: adoc/Private-Registry-Administration.adoc [hunk 1]
```diff
@@ -110,35 +110,251 @@ $ uaac client add suse_private_registry -s suseprivateregistrysupersecret --scop
...
+kubectl -n registry scale deployment -l component=core --replicas=0
+kubectl -n registry scale deployment -l component=notary-server --replicas=0
+kubectl -n registry scale deployment -l component=notary-signer --replicas=0
+----
+
+. change the password for the database server
++
+For an external database, use the means available from the public cloud provider to set a new admin password.
++
+For the internal database, this can only be done by accessing the database pod via `kubectl exec` and running a `psql` command to change the password, e.g.:
```
Comment (jsuchome) [inline adoc/Private-Registry-Administration.adoc:212]: That's not the _only_ way. You can e.g. use `kubectl port-forward` to forward the port from a service to your local machine and than access localhost with `psql`...

File: adoc/Private-Registry-Installation.adoc [hunk 2]
```diff
@@ -586,8 +578,6 @@ spec:
   sentinel:
     replicas: 3
     image: registry.suse.com/harbor/harbor-redis:2.1.1
-    customConfig:
-      - "dir /data"
```
Comment (jsuchome) [inline adoc/Private-Registry-Installation.adoc:590]: So why is this not needed after all?

File: adoc/Private-Registry-Administration.adoc (hunk 1 above)
Comment (stefannica) [reply adoc/Private-Registry-Administration.adoc:212]: @jsuchome that's true, but psql is already installed in the container, which makes it more convenient :)

File: adoc/Private-Registry-Installation.adoc (hunk 2 above)
Comment (stefannica) [reply adoc/Private-Registry-Installation.adoc:590]: @jsuchome this setting is only valid for the `redis` pods, not for the `sentinel` ones. Harmless, but this does cause some warnings to be logged in the sentinel pods.

File: adoc/Private-Registry-Administration.adoc (hunk 1 above)
Comment (jsuchome) [reply adoc/Private-Registry-Administration.adoc:212]: Sure, you do not have to cover all options, just do not mention the _only way_ is using `kubectl exec`

**Review by dirkmueller — APPROVED** [review]
skimmed over it. looks all good to me. thanks for all the incredible detail that you added there!

File: adoc/Private-Registry-Administration.adoc [hunk 3]
```diff
@@ -110,35 +110,251 @@ $ uaac client add suse_private_registry -s suseprivateregistrysupersecret --scop
 . Uncheck *UAA Verify Cert* if the UAA server uses a self-signed or untrusted certificate.
 . Click *Save* to complete the configuration.
 
-=== Registry Deployment Configuration Changes
+=== Registry Helm Configuration Changes
```
Comment (jsuchome) [inline adoc/Private-Registry-Administration.adoc:113]: Hm, this looks bit weird. I think _Registry Deployment Configuration Changes_ made sense, _Registry Helm Configuration Changes_ not that much ... I do not think that when you talk about configuration changes, you do not need to mention the deployment _method_ in the title.

Maybe just use _Registry Configuration Changes_.
Or _Post-deployment Changes_

File: adoc/Private-Registry-Administration.adoc [hunk 4]
```diff
@@ -110,35 +110,251 @@ $ uaac client add suse_private_registry -s suseprivateregistrysupersecret --scop
...
+
+==== Changing the Database Password
+[CAUTION]
+====
+Changing the database password is an administrative operation that has impact on service availability.
+====
+
+The password for the {spr} internal or external database service can be changed in three steps:
+
+. first, shutdown all {spr} services that use the database, to eliminate the risk of incomplete or failed transactions:
```
Comment (jsuchome) [inline adoc/Private-Registry-Administration.adoc:199]: Why do you start with lowercase here? (same with other bullet points)

**Review by jsuchome — CHANGES_REQUESTED** [review]

**Comment by r0ckarong** [comment]
@stefannica Please assign me to PRs when you add them and I need to review. I was not subscribed to this repository.

File: adoc/Private-Registry-Administration.adoc (hunk 3 above)
Comment (stefannica) [reply adoc/Private-Registry-Administration.adoc]: OK

**Comment by r0ckarong** [comment]
> @stefannica Please assign me to PRs when you add them and I need to review. I was not subscribed to this repository.

Jiri made me aware that you can't assign me without Admin rights apparently. Please just notify me via rocket chat then if your ready for review. Thanks.

File: adoc/Private-Registry-Administration.adoc (hunk 4 above)
Comment (stefannica) [reply adoc/Private-Registry-Administration.adoc]: Fixed

File: adoc/Private-Registry-Administration.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (stefannica) [reply adoc/Private-Registry-Administration.adoc]: Resolved, thank you, good point.

**Review by jsuchome — APPROVED** [review]

**Pull request merged**